src/test/java/io/swagger/swaggerhub/plugin/SwaggerHubClientTest.java

```diff
@@ -156,7 +158,9 @@ private SwaggerHubClient buildSwaggerHubClient(String basePath){
                 .scmPassword(SCM_PASSWORD)
                 .project(SCM_PROJECT)
                 .account(SCM_ACCOUNT)
-                .personalAccessToken(SCM_PERSONAL_ACCESS_TOKEN);
+                .personalAccessToken(SCM_PERSONAL_ACCESS_TOKEN)
+                .url(SCM_TFS_URL)
+                .projectCollection(SCM_TFS_PROJECT_COLLECTION);
```

Comment:
Remove references to TFS in the constant property names. We may have other SCM integrations in future which use those same properties